### PR TITLE
Don't load unused rails components

### PR DESCRIPTION
**Pull request by jlledom**
**What this PR does / why we need it**:

Part of the Rails 6 upgrade (https://github.com/3scale/porta/pull/3339)

Only load those rails components we actually use, and remove any reference to others.

**Which issue(s) this PR fixes** 

[THREESCALE-7405](https://issues.redhat.com/browse/THREESCALE-7405)

**Verification steps** 

Everything should work. Tests should pass.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,6 +1,32 @@
 require_relative 'boot'
 
-require 'rails/all'
+# frozen_string_literal: true
```
Comment (akostadinov) [inline config/application.rb:3]: this must be first line

Comment (akostadinov) [reply config/application.rb:3]: Otherwise looks great, thanks!

Comment (mayorova) [reply config/application.rb:3]: also `require_relative 'boot'` is repeated, so I think removing the first two lines will do it.

Comment (jlledom) [reply config/application.rb:1]: Fixed!

**Review by akostadinov — APPROVED** [review]

**Pull request merged**